db/views/materialized_patient_summaries_v01.sql

```diff
@@ -0,0 +1,118 @@
+SELECT p.recorded_at,
```

Comment:
This is intentionally formatted this way so that we can see all the exposed columns properly.